### PR TITLE
Cleaned up leader footstep ui in 'Member List' tile

### PR DESCRIPTION
**Pull request by prykon**
Responds to: #1398 

**Current behavior:**
- Black Footstep icon removed for leaders
- Leader toggle footstep icon is now black if member is leader

### CODE REVIEW

**Comment by corsacca** [comment]
Awesome work @prykon!

leader
The in between state is a bit confusing. 
@prykon what do you think of this idea?
When the foot is clicked: transform the icon into the loading circle. When the API call comes back, show the correct foot.

**Comment by prykon** [comment]
> Awesome work @prykon!
> 
> leader The in between state is a bit confusing. @prykon what do you think of this idea? When the foot is clicked: transform the icon into the loading circle. When the API call comes back, show the correct foot.

Sounds good! I'm on it...

**Comment by prykon** [comment]
The spinner appears when you make a member a leader. When you remove leadership, the row disappears before the spinner can be shown.

(screenshot frame rate slower than reality)

File: dt-groups/groups.js [hunk 1]
```diff
@@ -204,14 +207,15 @@ jQuery(document).ready(function($) {
...
   $(document).on("click", ".make-leader", function () {
+    $(this).children('i').attr('class', 'small')
+    let spinner = `<img src="/wp-content/themes/disciple-tools-theme/dt-core/admin/img/spinner.svg" width="15px">`
+    $(this).append(spinner)
     let id = $(this).data('id')
     let remove = false
     let existingLeaderIcon = $(`.member-row[data-id="${id}"] .leader`)
     if( window.lodash.find( post.leaders || [], {ID: id}) || existingLeaderIcon.length !== 0){
       remove = true
       existingLeaderIcon.remove()
```
Comment (corsacca) [inline dt-groups/groups.js:218]: @prykon  Remove `existingLeaderIcon.remove()` and the spinner will work both ways:

File: dt-groups/groups.js [hunk 2]
```diff
@@ -204,14 +207,15 @@ jQuery(document).ready(function($) {
     }
   })
   $(document).on("click", ".make-leader", function () {
+    $(this).children('i').attr('class', 'small')
+    let spinner = `<img src="/wp-content/themes/disciple-tools-theme/dt-core/admin/img/spinner.svg" width="15px">`
```
Comment (corsacca) [inline dt-groups/groups.js:211]: use this format: ``

Comment (corsacca) [reply dt-groups/groups.js:211]: spinner.svg is in the theme root.

**Review by corsacca — CHANGES_REQUESTED** [review]

**Comment by corsacca** [comment]
Very nice!

**Pull request merged**